subprojects/wrapper/src/main/java/org/gradle/wrapper/Install.java

```diff
@@ -71,18 +89,28 @@ public File call() throws Exception {
 
                 verifyDownloadChecksum(configuration.getDistribution().toString(), localZipFile, distributionSha256Sum);
 
-                logger.log("Unzipping " + localZipFile.getAbsolutePath() + " to " + distDir.getAbsolutePath());
                 unzip(localZipFile, distDir);
 
                 File root = getAndVerifyDistributionRoot(distDir, safeDistributionUrl.toString());
                 setExecutablePermissions(root);
                 markerFile.createNewFile();
 
+                printReleaseSummary();
+
                 return root;
             }
         });
     }
 
+    private void printReleaseSummary() {
+        if (gradleVersion != null) { // appVersion could be null in custom distributions
```

Comment:
:x: We should only render the message for GA versions. For RCs we should use https://gradle.org/release-candidate/ instead or not render it at all. Snapshots do not have an equivalent and should not render the message.